File: app/src/main/java/com/wlmtest/forecasttestjava/model/api/currentweatherdata/Wind.java
package com.wlmtest.forecasttestjava.model.api.currentweatherdata;

import com.squareup.moshi.Json;

public class Wind {

    @Json(name = "speed")
    private Double speed;
    @Json(name = "deg")
    private Double deg;
    @Json(name = "gust")
    private Double gust;

    public Double getSpeed() {
        return speed;
    }

    public void setSpeed(Double speed) {
        this.speed = speed;
    }

    public Double getDeg() {
        return deg;
    }

    public void setDeg(Double deg) {
        this.deg = deg;
    }

    public Double  getGust() {
        return gust;
    }

    public void setGust(Double  gust) {
        this.gust = gust;
    }

}

File: app/src/main/java/com/wlmtest/forecasttestjava/model/api/currentweatherdata/Wind.kt
package com.wlmtest.forecasttestjava.model.api.currentweatherdata

import com.squareup.moshi.Json

class Wind {

    @Json(name = "speed")
    var speed: Double? = null
    @Json(name = "deg")
    var deg: Double? = null
    @Json(name = "gust")
    var gust: Double? = null

}
